awscli/examples/ecr-public/batch-delete-image.rst

```diff
@@ -0,0 +1,92 @@
+**Example 1: To delete an image by using image digest ids, the image and all of its tags are deleted within a repository in a public registry**
+
+The following ``batch-delete-image`` example deletes an image by specifying the image digest.::
+
+    aws ecr-public batch-delete-image \
+        --repository-name project-a/nginx-web-app \
+        --image-ids imageDigest=sha256:b1f9deb5fe3711a3278379ebbcaefbc5d70a2263135db86bd27a0dae150546c2 
+
+Output::
+
+    {
+    "imageIds": [
+        {
+            "imageDigest": "sha256:b1f9deb5fe3711a3278379ebbcaefbc5d70a2263135db86bd27a0dae150546c2",
+            "imageTag": "latest"
+        }
+    ],
+    "failures": []
+    }
+
+For more information, see `Deleting an image in a public repository <https://docs.aws.amazon.com/AmazonECR/latest/public/public-image-delete.html>`__ in the *Amazon ECR Public User Guide*.
+
+**Example 2: To delete any image by specifying the tag associated with the image you want to delete from the repository.**
+
+The following ``batch-delete-image`` example deletes an image by specifying the tag associated with the image repository named ``project-a/nginx-web-app`` in a public registry. If you have only one tag and execute this command, it will remove the image. Otherwise, if you have multiple tags for the same image, specify one, and only the tag is removed from repositor and not the image. ::
```

Comment:
```suggestion
The following ``batch-delete-image`` example deletes an image by specifying the tag associated with the image repository named ``project-a/nginx-web-app`` in a public registry. If you have only one tag and execute this command, it will remove the image. Otherwise, if you have multiple tags for the same image, specify one, and only the tag is removed from repository and not the image. ::
```